google-ads/tests/context.py

```diff
@@ -0,0 +1,13 @@
+# -*- coding: utf-8 -*-
+
+import sys
+import os
+
+# Set working directory to parent so config.json can be found
+parent_dir = os.path.abspath(os.path.join(os.path.dirname(__file__), '..'))
+os.chdir(parent_dir)
```

Comment:
This seems an unusual approach at first glance. Can you please check how this is being done in other/the template integtrations.